getting-started/gopher-run/cmd/main.go

```diff
@@ -54,6 +64,46 @@ func main() {
 	log.Fatal(http.ListenAndServe(fmt.Sprintf(":%s", port), nil))
 }
 
+// Concatenate data from the top runs and start a Cloud ML training job on them
+func submitTrainingJob(a *app) {
+	// ctx := context.Background()
+	// bkt := a.bucket
+	// topPlayers, err := leaderboard.TopScores(ctx, a.fsClient)
+	// if err != nil {
+	// 	log.Printf("leaderboard.TopScores: %v", err)
+	// 	return
+	// }
+	// var appends [][]byte
+	// for _, player := range topPlayers {
+	// 	pld, err := bkt.Object("pldata/" + player.Name + "_pldata.csv").NewReader(ctx)
+	// 	if err != nil {
+	// 		log.Printf("NewReader: %v", err)
+	// 		continue
+	// 	}
+	// 	defer pld.Close()
+	// 	old, err := bkt.Object("pldata.csv").NewReader(ctx)
+	// 	if err != nil {
+	// 		log.Printf("NewReader: %v", err)
+	// 		continue
+	// 	}
+	// 	defer old.Close()
+	// 	b, err := ioutil.ReadAll(pld)
+	// 	if err != nil {
+	// 		log.Printf("ioutil.ReadAll: %v", err)
+	// 		continue
+	// 	}
+	// 	appends = append(appends, b)
+	// }
+	// var fullb []byte
+	// for _, item := range appends {
+	// 	fullb = append(fullb, item...)
+	// }
+	// new := bkt.Object("pldata.csv").NewWriter(ctx)
+	// defer new.Close()
+	// new.Write(fullb)
+	exec.Command("/bin/sh", "cmd/training.sh").Run()
```

Comment:
Would prefer to do this in Go directly -- `gcloud` probably isn't installed on Cloud Run or other environments (haven't checked).

Reply:
Will fix later